src/core/Akka.Cluster.Tests.MultiNode/QuickRestartSpec.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+using Akka.Util;
+
+namespace Akka.Cluster.Tests.MultiNode
+{
+    public class QuickRestartSpecConfig : MultiNodeConfig
```

Comment:
Used to verify that a cluster node who very quickly became unreachable, restarted, and then joined again can successfully rejoin the cluster